cmd/glasskube/cmd/bootstrap_git.go

```diff
@@ -155,10 +205,49 @@ var bootstrapGitCmd = &cobra.Command{
 			}
 		}
 
-		fmt.Fprint(os.Stderr, doneMessage)
+		// try to open argo-cd every 5 seconds until it succeeds in order to check whether it is ready
+		fmt.Fprintf(os.Stderr, "\nglasskube argo-cd application applied successfully!\n\nWaiting for argo-cd to be ready...")
+		ctx, cancel := context.WithTimeout(ctx, 5*time.Minute)
+		defer cancel()
+
+		tick := time.NewTicker(5 * time.Second)
+		tickC := tick.C
+		defer tick.Stop()
+	WaitLoop:
+		for {
+			select {
+			case <-ctx.Done():
+				fmt.Fprintf(os.Stderr, "\nargo-cd has not become ready in the specified timeout.\n")
+				break WaitLoop
+			case <-tickC:
+				if res, err := open.NewSilentOpener().Open(ctx, argoCdPkg, "", "localhost", 0); err == nil {
+					res.Stop()
+					break WaitLoop
+				}
+				// else: generally assume an error means not ready yet
+			}
+		}
+
+		// get initial admin password to print it
+		var adminPwStr string
+		if adminPw, err := clientset.CoreV1().Secrets(argocdNamespace).
+			Get(ctx, "argocd-initial-admin-secret", metav1.GetOptions{}); err != nil {
+			fmt.Fprintf(os.Stderr, "\nCould not obtain initial ArgoCD password: %v\n\n", err)
+		} else {
+			adminPwStr = string(adminPw.Data["password"])
+		}
+
+		fmt.Fprintf(os.Stderr, doneMessage, adminPwStr)
 	},
 }
 
 func init() {
-	bootstrapGitCmd.Flags().StringVar(&bootstrapGitCmdOptions.url, "url", "", "URL of the GitOps Repository")
+	bootstrapGitCmd.Flags().StringVar(&bootstrapGitCmdOptions.url, "url", "",
+		"URL of the GitOps Repository")
+	bootstrapGitCmd.Flags().StringVar(&bootstrapGitCmdOptions.branch, "branch", "",
+		"Branch of the GitOps Repository to use (default is main)")
+	bootstrapGitCmd.Flags().StringVar(&bootstrapGitCmdOptions.username, "username", "",
+		"Username to use for authentication")
+	bootstrapGitCmd.Flags().StringVar(&bootstrapGitCmdOptions.token, "token", "",
```

Comment:
A lot of tools also support the `GITHUB_TOKEN` environment variable. I think we should do the same.